.goreleaser.yaml

```
@@ -0,0 +1,66 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  - main: ./cmd/celestia-appd
+    binary: celestia-appd
+    env:
+      - CGO_ENABLED=0
+      - SDKPath={{ "github.com/cosmos/cosmos-sdk/version" }}
+    goos:
+      - linux
+      - windows
+      - darwin
```

Also I would limit this to 64-bit architectures.  
```suggestion
      - darwin
    goarch:
      - amd64
      - arm64
```
Unless we test the i386 arch on CI as well.

---

[informational] I think we only have tests on ubuntu-latest which is 64 bit

Ref:

https://github.com/celestiaorg/celestia-app/blob/63915bf4cf32772fe75edf3e99df48315007aec6/.github/workflows/test.yml#L23

- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners
- https://releases.ubuntu.com/jammy/